src/contributing/index.md

```diff
@@ -0,0 +1,95 @@
+# Contributing
+
+Documentation is critical to making a usable data platform.
+When surveying our users,
+their most common complaint has been our lack of documentation.
+It's important that we improve our documentation as often as possible.
```

Comment:
To:
Documentation represents a critical ingredient when creating a usable data platform.
When Mozilla users are surveyed, one of the most common complaints deals with
the lack of available documentation. Therefore, it is important to improve the 
documentation as often and frequently as possible.

```